### PR TITLE
Port to python 3, GTK 3, sugargame v1.3

### PR DESCRIPTION
**Pull request by sparshg**
@chimosky @srevinsaju
I haven't tested the activity in python 2 yet due to some error in my setup, I think most of the porting is done though.

### CODE REVIEW

File: activity.py [hunk 1]
```diff
@@ -1,54 +1,57 @@
...
-        self.box.show()
-        self.set_canvas(self.box)
+        watch = Gdk.Cursor.new(Gdk.CursorType.WATCH)
+        self._set_cursor(watch)
 
         def run():
             import game
             spyral.director.init((0,0), fullscreen=False, max_fps=30)
             game.main(self)
             spyral.director.run(sugar = True)
```
Comment (chimosky) [inline activity.py:45]: Could you move this to below the assignments below?

File: activity/activity.info [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [Activity]
 name = Colgadito
-activity_version = 3
+activity_version = 4
```
Comment (chimosky) [inline activity/activity.info:3]: This isn't a PR for a new release so I'm guessing this wasn't intentional, please leave the change till you're making a release.

File: init.py [hunk 3]
```diff
@@ -42,11 +42,11 @@
...
 
 if not os.path.isfile("activity/activity.svg"):
-    print 'No icon found. A placeholder has been copied to activity/activity.svg, you\nshould replace this with your own icon.'
+    print('No icon found. A placeholder has been copied to activity/activity.svg, you\nshould replace this with your own icon.')
     f = 'skel/activity.svg'
     shutil.copy(f, 'activity/activity.svg')
     
-print 'All done.'
\ No newline at end of file
+print('All done.')
```
Comment (chimosky) [inline init.py:52]: Also remove the `No newline at end of file`.

**Comment by chimosky** [comment]
Reviewed, not tested. 

Could you share the error you got while trying to test in python 2?

**Comment by sparshg** [comment]
Seeing the issue #1 how should we proceed?

Also I meant there was some unrelated error in running python 2 activities with native sugar, I will try using live build instead.

**Comment by chimosky** [comment]
You can re open this PR in that repo then, I'll delete this one.

**Pull request closed without merging**